AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Elevation-out-of-range.rst

```diff
@@ -0,0 +1,124 @@
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Carlos Morales, Erik Dennihy*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-AuxTel-Troubleshooting-General-Troubleshooting-AuxTel-AuxTel-Elevation-out-of-range:
+.. Each section should include a label for cross-referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with a reST object such as a title or figure, you must include the link with an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+######################################
+AuxTel Elevation Out of range
+######################################
+
+.. _AuxTel-Elevation-out-of-range-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this AT correct_pointing failure procedure needs to be used. Describe the symptoms that the user sees to use this procedure.
+
+This procedure should be used when the AuxTel :file:`auxtel/correct_pointing.py` script fails with the error *Rejected: elevation out-of-range*.
+
+.. _AuxTel-Elevation-out-of-range-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide a simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+- The :file:`auxtel/correct_pointing` script fails with the following error traceback:
+
+    .. code-block:: text
+       
+       lsst.ts.salobj.base.AckError: msg='Command failed', 
+       ackcmd=(ackcmd private_seqNum=942292490, ack=<SalRetCode.CMD_FAILED: -302>, 
+       error=6611, result='Rejected : elevation out of range')
+
+- This occurs when the target search within the :file:`correct_pointing.py` script is not successful in the specified location in the sky within the `mag_limit`, `mag_range`, and `radius`.
+
+.. _AuxTel-Elevation-out-of-range-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. _AuxTel-Elevation-out-of-range-Step-1:
+
+To resolve the *Rejected: elevation out-of-range* error in the :file:`correct_pointing` script, follow these steps:
```

Comment:
Error in code-block

Reply:
or in " marks (confirm with Iona) because italics is used for specific situations